### PR TITLE
Mpgjon patch 4

### PR DESCRIPTION
**Pull request by mpgjon**
Working on issues in the project that include:

- Removing Prospector from Software page
- Adding in a Publications section in main menu and initial .md file
- Adding Favicon asset
- Adding new logo asset in Media directory to fix broken link

### CODE REVIEW

**Comment by klumb** [comment]
Thank you @mpgjon

**Comment by klumb** [comment]
@mpgjon, please add DCO signoff.

**Comment by mpgjon** [comment]
I downloaded a local Git client and run the DCO force command in Terminal, then pushed back to the repo.  Did that actually work?!

**Comment by klumb** [comment]
@mpgjon, it didn't work. Add the signoff to the comment and I will set DCO to pass.

**Comment by mpgjon** [comment]
DCO signoff via comment until I figure out how to do that on my own.. sorry y'all.

Signed-off-by: Jon Lawrence <jon@mpgstudios.com>

**Comment by klumb** [comment]
No worries, see my comments on the other PR. Adding the flag to the commit is pretty painless for future PRs.

**Comment by klumb** [comment]
Also, please add a description to the PR. It will help with the review.

**Comment by GeorgLink** [comment]
@mpgjon thanks for suggestion all these changes.

As a general rule, please create small pull requests that only make one focused change. Adding a favicon is one pull request candidate. Removing prospector.md is one pull request. Fixing a broken link is yet another pull request. ... you get the idea.

Separating out pull requests allows the community to review and approve changes more easily and one change would not hold up other improvements.

Regarding DCO: @klumb we could "Squash-merge" all commits, then we only need one sign-off that get's included in the final commit message.

File: About/Publications/PublishedWorks.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Signed-off-by: Jon Lawrence <jon@mpgstudios.com>
```
Comment (GeorgLink) [inline About/Publications/PublishedWorks.md]: Please remove the sign-off from the file. DCO sign-offs need to be part of the commit message.

**Pull request closed without merging**